src/control/XournalMain.cpp

```diff
@@ -383,10 +383,16 @@ using XMPtr = XournalMainPrivate*;
 
 /// Checks for input method compatibility and ensures it
 void ensure_input_model_compatibility() {
+    char* unsupportedImModules[] = {"xim", "gcin"};
     const char* imModule = g_getenv("GTK_IM_MODULE");
```

Comment:
I would be more comfortable with this if we used `std::string` here. :)

E.g.
```c++
const char* imModuleEnv = g_getenv("GTK_IM_MODULE");

if (imModuleEnv != nullptr) {
  std::string imModule { imModuleEnv };
  if (imModule == ... || imModule == ...) {
   ...
  }
}
```

Reply:
Updated. Thanks for your comment!